extractor/src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInName/Configs/config-all-examples-in-one.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name = "Checker">
+    <property name="charset" value="UTF-8"/>
+
+    <!-- do not change severity to 'error', as that will hide errors caused by exceptions -->
+    <property name="severity" value="warning"/>
+
+    <!-- haltOnException is required for exception fixes and reporting of all exceptions -->
+    <property name="haltOnException" value="false"/>
+
+    <!-- BeforeExecutionFileFilters is required for sources of java9 -->
+    <module name="BeforeExecutionExclusionFileFilter">
+        <property name="fileNamePattern" value="module\-info\.java$" />
+    </module>
+
+    <module name="TreeWalker">
+        <module name="AbbreviationAsWordInName">
+            <property name="id" value="example1"/>
+        </module>
+
+        <module name="AbbreviationAsWordInName">
```

Comment:
This should be expected file, all other Examplex are input files

Reply:
Okay, so far with my testcases, i give one input file say Example2.java. And, when i run the test, it generates a generated-config-example2.java for me. But, if i understood you right, you want all the 7 examples to be given as input and generate a final all-in-one-config.xml. But, do i have to write a test case for that?

or, the review is just to change the name?

Reply:
As you place few files in resources and give path to folder, code should generate bundles for each example and all-in-one.

Your test will be kind of launcher and few statements to assert generated results with expected filees.